media-sound/setbfree/setbfree-9999.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+inherit toolchain-funcs multilib
+
+DESCRIPTION="MIDI controlled DSP tonewheel organ"
+HOMEPAGE="http://setbfree.org"
+
+if [ "${PV}" = "9999" ]; then
+	inherit subversion git-r3
+	EGIT_REPO_URI="https://github.com/pantherb/setBfree.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/pantherb/setBfree/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="convolution"
+
+RDEPEND="media-sound/jack-audio-connection-kit
+	>=media-libs/alsa-lib-1.0.0:=
+	media-libs/liblo:=
+	media-libs/lv2
```

Comment:
Is it not possible to choose what to build (standalone application or lv2 plugin)/